jobs/github_org_repos.rb

```diff
@@ -0,0 +1,11 @@
+require 'dashing'
+
+# This job runs once a day to store the list of all GitHub repos for an org
+# in an env var. That way, it can be reused by other widgets without having
+# to make more API calls.
```

Comment:
It'd be nice if there was a way to trigger this via an org webhook, to add new repos as they happen to appear, and remove old ones if they go away. Obviously a future enhancement, but do you know if that's possible?